packages/scalars/package.json

```diff
@@ -49,15 +49,15 @@
       "import": "./dist/es/src/scalars/src\\scalars\\index.js",
       "types": "./dist/types/src/scalars/src\\scalars\\index.d.ts"
     },
-    "./EmailAddress": {
-      "require": "./dist/cjs/src/scalars/EmailAddress.js",
-      "import": "./dist/es/src/scalars/EmailAddress.js",
-      "types": "./dist/types/src/scalars/EmailAddress.d.ts"
+    "./src\\scalars\\EmailAddress": {
```

Comment:
Could you clarify if this change was needed for compatibility or if it addresses a specific issue?

Reply:
It was an unwanted change. I rolled it back

Reply:
LGTM